src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+import { ADD_BOOK, REMOVE_BOOK } from '../../data/constants';
+
```

Comment:
- [ **OPTIONAL** ] To correctly follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), I think it would be better if you the concerned `books` actions type from the `'../../data/constants'` file and come define them here by following the `ducks pattern` approach as I showed an example in the previous comment.